packages/accounts-2fa/2fa-server.js

```diff
@@ -0,0 +1,123 @@
+import { Accounts } from 'meteor/accounts-base';
+import twofactor from 'node-2fa';
+import QRCode from 'qrcode-svg';
+import { Meteor } from 'meteor/meteor';
+
+Accounts.is2faEnabledForUser = selector => {
+  if (!Meteor.isServer) {
+    throw new Meteor.Error(
+      400,
+      'The function is2faEnabledForUser can only be called on the server'
+    );
+  }
+
+  if (typeof selector === 'string') {
+    if (!selector.includes('@')) {
+      selector = { username: selector };
+    } else {
+      selector = { email: selector };
+    }
+  }
+
+  const user = Meteor.users.findOne(selector) || {};
+  const { twoFactorAuthentication } = user;
+  return (
+    twoFactorAuthentication &&
+    twoFactorAuthentication.secret &&
+    twoFactorAuthentication.type === 'otp'
+  );
+};
+
+Accounts.generate2faToken = secret => twofactor.generateToken(secret);
+
+Accounts.isTokenValid = (secret, token) => {
```

Comment:
Why is this method exposed in the Accounts object?

Reply:
Because I also need to called inside the accounts-password to verify if the code provided is valid https://github.com/meteor/meteor/blob/426872ebb4fe00c14914a66f5f4705d3d88823be/packages/accounts-password/password_server.js#L195

Reply:
If this is only for internal usage we should name it starting with `_` so we can change it later without notifying everybody or considering a breaking change.